boot/reboot.go

```diff
@@ -0,0 +1,101 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2022 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package boot
+
+import (
+	"fmt"
+	"os/exec"
+	"time"
+
+	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/osutil"
+)
+
+func getRebootArgsPathRuntime() string {
+	return "/run/systemd/reboot-param"
+}
+
+// GetRebootArgsPath is used so we can mock the path easily in tests
+var GetRebootArgsPath = getRebootArgsPathRuntime
```

Comment:
I'm not sure why this is a function instead of a string, also it shouldn't be exported, we either mock things for the package in export_test.go  or we have a Mock* helper that is exported if we need to mock something globally